public/scripts/main.js

```diff
@@ -2,31 +2,95 @@ var __connected_ft = (function(){
 
 	'use strict';
 
-	var API_KEY = 'AAAARkLBNBk:APA91bFKrf-HAIZx7uVPP2fEOwLkoGi_EWpjuo9i-yMqg5Ub8Dg3p0MafUtloIvXDpZAJ48ZpftalWrPF7GgjebKYJf7zNiOmnu78CpTAXn2pdIEZ1oTd4JZnAibNarHOB_UNm1qOFGK';
+	var API_KEY = 'AAAARkLBNBk:APA91bG-5SRR12484VbE3rNlR1Xr2N0OunZNhq9YFpn38s_8mpXnqpCoTUc9MqF_qgrFitrc-pQox8pMb4C6RPprEp2KiAH7L3ET2los9em7n-6-hgAbjUOtpqudkrSvfzDFtw3tmJZh';
```

Comment:
Client-side GCM key for notifications. #3 removes keys from the repo. This key is now revoked.